.github/workflows/CI.yaml

```diff
@@ -245,6 +245,11 @@ jobs:
       - name: "Test: Runtime upgrade and queue processing"
         run: ./scripts/gear.sh test runtime-upgrade
 
+      - name: "Test: Runtime benchmarks work"
+        run: |
+          cargo build -p gear-node --release --features runtime-benchmarks
```

Comment:
I'd add the feature to `./gear.sh build --features runtime-benchmarks` to avoid rebuild of deps

Reply:
not all our workspace members have such feature: pallets + node only 
it also a bit changes logic of execution: adds new extrinsics and increases recursion limit
so I vote for rebuild for this case atm

Reply:
> not all our workspace members have such feature: pallets + node only

Which is ok to cargo

> it also a bit changes logic of execution: adds new extrinsics and increases recursion limit

So what's the execution logic change?

Reply:
I see we do something similar with `pallet-gear` and your change adds only 3 minutes, so it's ok